lib/init.js

```diff
@@ -15,17 +17,26 @@ init.usage = usage(
   'init',
   '\nnpm init [--force|-f|--yes|-y|--scope]' +
   '\nnpm init <@scope> (same as `npx <@scope>/create`)' +
-  '\nnpm init [<@scope>/]<pkg> (same as `npx [<@scope>/]create-<pkg>`)'
+  '\nnpm init [<@scope>/]<name> (same as `npx [<@scope>/]create-<name>`)' +
+  '\nnpm init <pkg> (same as `npx <pkg>`)'
```

Comment:
This line is kinda confusing, tbh. I'm not really sure at first glance how it might be different than the previous one (since it's not clear this is about non-registry packages).

Reply:
I think I saw `<pkg>` used in another place to mean `<folder|tarball|git url|user/repo>`.
Maybe a way to say everything else is just forwarded to `npx`.